elementor.php

```diff
@@ -105,3 +105,33 @@ function elementor_fail_wp_version() {
 
 	echo wp_kses_post( $html_message );
 }
+
+// TODO: Remove when the minimum required WordPress version is 6.5 or higher.
+if ( ! function_exists( 'wp_enqueue_script_module' ) ) {
+	function wp_enqueue_script_module( $handle, $src = '', $dependencies = [], $version = false ): void {
+		$bc_handle = '__elementor_wp_bc_' . $handle;
+
+		if ( ! wp_script_is( $bc_handle, 'registered' ) ) {
+			wp_register_script_module( $bc_handle, $src, $dependencies, $version );
+
+		}
+
+		wp_enqueue_script( $bc_handle );
+	}
+}
+
+// TODO: Remove when the minimum required WordPress version is 6.5 or higher.
+if ( ! function_exists( 'wp_register_script_module' ) ) {
+	function wp_register_script_module( $handle, $src, $dependencies = [], $version = false ): void {
+		$bc_handle = '__elementor_wp_bc_' . $handle;
+
+		wp_register_script( $bc_handle, $src, $dependencies, $version, true );
+
+		add_filter( 'script_loader_tag', function ( $tag, $script_handle ) use ( $bc_handle ) {
+			if ( $script_handle === $bc_handle ) {
+				$tag = str_replace( '<script ', '<script type="module" ', $tag );
+			}
+			return $tag;
+		}, 10, 2 );
+	}
+}
```

Comment:
Copied from core?

Reply:
It is not copied. WordPress has created an additional class for this. This is a simplified version to support module script enqueues.